source/HockeySDK.AndroidBindings/Additions/CrashManager.cs

```diff
@@ -73,8 +73,13 @@ private static void ConnectUnhandledExceptionEvents(global::HockeyApp.Android.Cr
 
 				AndroidEnvironment.UnhandledExceptionRaiser += (sender, e) => TraceWriter.WriteTrace(e.Exception);
 				AppDomain.CurrentDomain.UnhandledException += (sender, e) => TraceWriter.WriteTrace(e.ExceptionObject);
-				TaskScheduler.UnobservedTaskException += (sender, e) => TraceWriter.WriteTrace(e.Exception, terminateOnUnobservedTaskException);
-
+				TaskScheduler.UnobservedTaskException += (sender, e) =>
+				{
+						if (!e.Observed)
```

Comment:
Isn't that check always false? Should we approve #112 instead?

Reply:
No. This one looks better to me. It uses a predefined standard api and allows filter exceptions individually.

Reply:
I agree with Ivan here.